Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -10724,7 +10724,7 @@ def get_device_state(topology: Topology, target: str):
     @staticmethod
     def get_ha_status(
         topology: Topology, device_filter_str: Optional[str] = None, target: Optional[str] = None
-    ) -> List[ShowHAState]:
+    ) -> List[ShowHAState] | ShowHAState:
```

Comment:
why return 1 instead of a list of 1? Lets be consistent with our return types

Reply:
This is how it was originally written by @adambaumeister, the reason was related to the way the context shows the data, i don't recall exactly why, if we change it that might affect the way context is showing the data, so i prefer not to change it if it was implemented that way in the first place.

I had the same question here: https://github.com/demisto/content/pull/17689/files#r808922114 (you can read)